gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -102,8 +102,8 @@ export default function NxGlobalSidebarPage() {
                 <NxTable.Cell>
                   The path to an image placed in the top left corner which displays branding and product name.
                   <NxWarningAlert>
-                    Note: the image should not be larger than 145px wide and 32px tall. It should
-                    display either a logo (apx 32px x 32px) and the product name, or just the product name.
+                    Note: the image should not be larger than 155px wide and 34px to 43px tall. It should
```

Comment:
34? The previous max was 32 so I'd think we'd want our min to be no larger than 32.  But further, why do we even need a min? We didn't before.

Reply:
> 34? The previous max was 32 so I'd think we'd want our min to be no larger than 32. But further, why do we even need a min? We didn't before.

Confirmed with Jason, and no minimum height is needed. Removed here: [9408439](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/940843939654746264321c4a310af4e69db68282). I will update the documentation once they clarify the padding between the logo & button

Reply:
> > 34? The previous max was 32 so I'd think we'd want our min to be no larger than 32. But further, why do we even need a min? We didn't before.
> 
> Confirmed with Jason, and no minimum height is needed. Removed here: [9408439](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/940843939654746264321c4a310af4e69db68282). I will update the documentation once they clarify the padding between the logo & button

Updated to remove min-width, and calculated the available space, which is 158px wide: [00198e7](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/00198e73dd714936edc3eab073ca5ca4ddc4f47c)